Sources/GoogleAI/GenerativeModel.swift

```diff
@@ -42,7 +42,8 @@ public final class GenerativeModel {
   /// Initializes a new remote model with the given parameters.
   ///
   /// - Parameters:
-  ///   - name: The name of the model to be used, e.g., "gemini-pro" or "models/gemini-pro".
+  ///   - name: The name of the model to be used, e.g., `"gemini-pro"` or `"models/gemini-pro"`; see
```

Comment:
Should we recommend just one of `"gemini-pro"` or `"models/gemini-pro"`?

Reply:
IMHO, we should just recommend `"models/gemini-pro"` since it can be copy-pasted from the docs (updated the PR) and it'll keep things simpler in a future world with more types of models (less need to explain that `"gemini-pro"` gets prefixed to `"models/gemini-pro"`).

I also just noticed that in [Gemini models](https://ai.google.dev/models/gemini) "models/gemini-pro" is called a "Model code", not a "Model name". I wonder if we should cover our bases and say "The *code* name of the model to be used". Thoughts?

Reply:
Discussed across teams, updated to `"gemini-pro"` without the `models/` prefix.